docs/manual/manual.adoc

```diff
@@ -996,6 +996,36 @@ the `--fetch-remote-resources` option to automatically download it using the
  ...
 
 
+=== Evaluating XCCDF rules with multiple checks
+
+Normally, each XCCDF rule references to a single check with a specified name.
+However, if @name attribute of xccdf:check-content-ref of a given rule is omitted,
+multiple checks can be executed to evaluate the rule.
+This is often used in rules verifying that all security patches are installed.
+In such case, only a single result is produced for a XCCDF rule by default.
+The result is computed from all the checks from the content in the referenced location.
+But using this approach doesn't give the expected user experience.
+Instead, it would be better to have separate result for each check in the results document.
```

Comment:
Nitpicks: Maybe more generical here and don't add judgement value (better).

`The default approach might not be aligned with expected behaviour.`